docs/deployment/kernel-tuning/README.md

```diff
@@ -58,6 +58,8 @@ echo madvise | sudo tee /sys/kernel/mm/transparent_hugepage/enabled
 
 Also, set `transparent_hugepage=madvise` on your kernel command line (e.g. in /etc/default/grub) to persistently set this value.
 
+All these system level tunings are conveniently packaged in [shell script](https://github.com/minio/minio/blob/master/docsk/deployment/kernel-tuning/sysctl.sh). Please review the shell script for our recommendations.
```

Comment:
Can you fix the doc link - `https://github.com/minio/minio/blob/master/docs/deployment/kernel-tuning/disk-tuning.sh`

Reply:
```suggestion
All these system level tunings are conveniently packaged in [shell script](https://github.com/minio/minio/blob/master/docs/deployment/kernel-tuning/sysctl.sh). Please review the shell script for our recommendations.
```